common/lib/xmodule/xmodule/templates/about/overview.yaml

```diff
@@ -41,7 +41,8 @@ data: |
         <h2>Frequently Asked Questions</h2>
         <article class="response">
           <h3>What web browser should I use?</h3>
-          <p>The Open edX platform works best with current versions of Chrome, Edge, Firefox, Internet Explorer, or Safari.</p>
+          <p>We recommend the latest version of Chrome or Firefox.</p>
+          <p>The Open edX platform also supports the latest versions of Edge, Internet Explorer, and Safari.</p>
```

Comment:
The Open edX platform also supports the latest versions of Edge and Safari.

Reply:
I would just leave the sentence as is and drop IE from the list. The tiering of Chrome / Firefox above Edge + Safari seems arbitrary to me.

Reply:
Sounds good, I will open a PR to https://github.com/edx/edx-documentation/blob/master/en_us/shared/browsers.rst as well and revert the language as you suggested unless others have concerns.